lib/dealership.rb

```diff
@@ -0,0 +1,54 @@
+class Dealership
+  attr_reader :dealer_name, :address, :inventory
+  def initialize(dealer_name, address)
+    @dealer_name = dealer_name
+    @address = address
+    @inventory = []
+  end
+
+  def inventory_count
+    @inventory.length
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+    @inventory.length > 0
+  end
+
+  def cars_by_make(make)
+    @inventory.select {|car_make| car_make.make == make}
```

Comment:
Since you are iterating through the cars via `@inventory` rather than the makes, I might change the variable.

```
@inventory.select {|car| car.make == make}
```